python-shell/README.md

```diff
@@ -1,6 +1,13 @@
-# Deprecated
+# README
+
+Copyright 2016 Crown Copyright
```

Comment:
Is the plan not to update the header as we're only reinstating things not strictly changing?

Reply:
Yes this PR is strictly a git revert, but you're right that date does need changing. I also don't like the fact the readme starts with a license when there is a separate license file.

Reply:
It feels like we need to rework the way the license stuff is managed more generally. Do these kind of files really need a header? For example, we don't have a header on every page of the docs.

Reply:
Yeah I am not sure, seems unnecessary and makes it harder to read for a readme.